docker-compose.yml

```diff
@@ -0,0 +1,81 @@
+version: '3'
+
+services:
+
+  elasticsearch1:
+    image: docker.elastic.co/elasticsearch/elasticsearch:5.3.1
+    container_name: elasticsearch1
+    ports:
+      - 9200:9200
+      - 9300:9300
+    environment:
+      cluster.name: "docker-cluster"
+      bootstrap.memory_lock: "true"
+      ES_JAVA_OPTS: "-Xmx256m -Xms256m"
+      # disable X-Pack
+      # see https://www.elastic.co/guide/en/x-pack/current/xpack-settings.html
+      #     https://www.elastic.co/guide/en/x-pack/current/installing-xpack.html#xpack-enabling
+      xpack.security.enabled: "false"
+      xpack.monitoring.enabled: "false"
+      xpack.graph.enabled: "false"
+      xpack.watcher.enabled: "false"
+    ulimits:
```

Comment:
Do we really need all this memory stuff?  It feels more like stuff that should be added for the production deployment rather than the ease of deployment file?